zingoconfig/src/lib.rs

```diff
@@ -506,6 +506,13 @@ pub enum ChainType {
     Mainnet,
 }
 
+impl Default for ChainType {
```

Comment:
My gut would expect a default ChainType to be Mainnet. This (and by extension, the default zingoconfig) strikes me as counterintuitive.